### ISSUE TITLE
TypeError: '<' not supported between instances of 'str' and 'int'

### ISSUE THREAD

**Issue by wuyue0509** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#301]
![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/124325018/4e106507-5e7a-4815-8b0b-670d75fc28a4)

**Comment by AbdullahAlfaraj** [comment]
I need more info. please provide: 
- Plugin version
- Method of installation (1,2, or 3)
- ControlNet version

**Comment by wuyue0509** [comment]
Method 2，
![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/124325018/92dd391f-6334-445a-91f1-dca35901f187)

![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/124325018/ebcac2e1-c840-43bc-aad3-e587b19f5299)

**Comment by darkimage** [comment]
same issue here with method 3 and hash 99408b9 of control net extension. i also tried other hashes (controlnet) but same results

**Comment by an303042** [comment]
Same for me:
```
Error running process: F:\SD\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "F:\SD\stable-diffusion-webui\modules\scripts.py", line 451, in process
    script.process(p, *script_args)
  File "F:\SD\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 751, in process
    if unit.threshold_a < 0:
TypeError: '<' not supported between instances of 'str' and 'int'
```

Plugin V 1.2.5
Installation method 3
ControlNet - v1.1.220

Pretty sure it ran errorless 24 hours ago.

Oh, and thanks for this amazing work!

**Comment by AbdullahAlfaraj** [comment]
This is the latest version of ControlNet that I’ve tested the plugin with. You might want to use it until I have time to test the new version. 
![image](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/7842232/8195acc6-4004-4841-a7a2-af62b89324c5)

**Comment by wuyue0509** [comment]
thank you and wait for the next Version

**Comment by AbdullahAlfaraj** [comment]
fixed in v1.3.0
